docs/source/executorch-arm-delegate-tutorial.md

```diff
@@ -316,7 +327,43 @@ python3 -m examples.arm.aot_arm_compiler --model_name="add" --delegate
 # should produce ./add_arm_delegate.pte
 ```
 
-At the end of this, we should have two different `.pte` files. First one with the [SoftmaxModule](#softmaxmodule), without any backend delegates. And the second one with the [AddModule](#addmodule), and with Arm Ethos-U backend delegate enabled. Now let's try to run these `.pte` files on a Corstone-300 platform in a bare-metal environment.
+### Delegated Quantized Workflow
+Before generating the `.pte` file for delegated quantized networks like MobileNetV2, we need to build the `quantized_ops_aot_lib`
```

Comment:
Nit: Is this covered by some other docs? If not does it make sense to create another page and link it from here? Rationale is if other use cases need to do this as well.

Reply:
> Is this covered by some other docs?
No from my understanding. That makes sense. I will refactor this part once we got more use cases. I will leave this as this right now.